bootloader/assets/data/grub.cfg

```diff
@@ -5,9 +5,9 @@ set timeout=3
 set timeout_style=hidden
 
 # load only kernel_status from the bootenv
-load_env --file /EFI/ubuntu/grubenv kernel_status
+load_env --file /EFI/ubuntu/grubenv kernel_status snapd_extra_cmdline_args
 
-set cmdline="console=ttyS0 console=tty1 panic=-1"
+set snapd_static_cmdline_args='console=ttyS0 console=tty1 panic=-1'
```

Comment:
I know this definition comes from the original configuration file, however when the command line was defined entirely in gadget it would be easy for a vendor to change it to suit their needs. What should the device manufacturer do when they need to remove something from the static cmdline, e.g. to disable the serial console in a device using ttyS0 for a different purpose?

Reply:
I guess IMHO, the console bits should be part of the gadget defined portion of the kernel command line (i.e. "extras"), and panic=-1 would be the part that is "static", but I think since we have to obey the original ordering of the kernel command line as written for the beta (until resealing is a thing), we should probably not change this yet and carefully manage the transition of ownership of console= from static to gadget

Reply:
Yes, Ian assessment is correct. That's why we do need a way for the gadget to contribute to the command line. The issue there is mostly a matter of finding the right/agreeable syntax. We probably should progress more on resealing in general before worrying about that though. For now keeping the command line as is, is the least confusing path.

Reply:
On top of that some kernel command line options have different semantics than others. Eg. `panic=<n>` can be specified a number of times, and the last occurrence actually sets the value, on the other hand `console=<s>` is somewhat peculiar as you can specify it multiple tiles, but IIRC in the end there will be one console per device type (one console for serial devices, one for vga devices and so on).